File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/cryo-lord.py
texto = ""
print("@-------------------@")
for i in range(1,101):
    if i%3 == 0:
        texto += "fizz"
    if i%5 == 0:
        texto += "buzz"
    if texto != "":
        print(f"     {i}: " + texto + "\n@-------------------@")
        texto = ""
    else:
        print(f"         {i}" + "\n@-------------------@")

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/java/Cryo-Lord.java
class Cryo_lord
{
    public static void main(String[] args)
    {
        var text = "";
        System.out.println("----  Iniciando ciclos  ----");
        for (int i=0; i < 101; i++)
        {
            if ((i % 3) == 0)
            {
                text += "fizz";
            }
            if ((i % 5) == 0)
            {
                text += "buzz";
            }
            //--------------------------------------------//
            if (!(text == ""))
            {
                System.out.println("----  "+ i + " " + text +"  ----");
            }
            else
            {
                System.out.println("----  " + i + "  ----");
            }
            text = "";
        }
        System.out.println("----  Ciclo terminado  ----");
    }    
}